cryptol-remote-api/python/cryptol/quoting.py

```diff
@@ -29,7 +29,8 @@ def to_cryptol_str(val : Union[CryptolValue, str, CryptolJSON]) -> str:
     elif isinstance(val, OpaqueValue):
         return str(val)
     elif isinstance(val, str):
-        return f'"{val}"'
+        val_san = val.replace('\\', '\\\\').replace('"', '\\"').replace('\n', '\\n')
```

Comment:
Should we include all of Python's escape characters here for completeness? https://python-reference.readthedocs.io/en/latest/docs/str/escapes.html

Reply:
In this case we're sanitizing the string for Cryptol, not for Python, so I believe backslash, double-quote, and newline should be enough? https://github.com/GaloisInc/cryptol/blob/master/src/Cryptol/Parser/Lexer.x#L77

Reply:
Oops, I misunderstood the problem! Indeed, what you have should be enough, but do leave a reference to the Cryptol lexer as justification for this choice.

Reply:
Done - thanks!